janusgraph-berkeleyje/src/main/java/org/janusgraph/diskstorage/berkeleyje/BerkeleyJEStoreManager.java

```diff
@@ -110,6 +110,7 @@ private void initialize(int cachePercent) throws BackendException {
             envConfig.setAllowCreate(true);
             envConfig.setTransactional(transactional);
             envConfig.setCachePercent(cachePercent);
+            envConfig.setSharedCache(true);
```

Comment:
(nitpick) If you like, you could a config to deactivated shared cache in a certain case.

Reply:
ok